### ISSUE TITLE
New Institution type suggestion

### ISSUE THREAD

**Issue by wujastyk** [PanditProject/Website#42]
Could "Illam" be added to the list of possible "Institution type" (when creating an Institution).

Maybe Illam is covered by "House", but I think it's worth it's own entry.  Illams are very unique

**Comment by ybronner** [comment]
I agree that we don't want to lose important information such as Mana and Illam as terms of art in local culture and as things we can search for. The question is how to go about it. I'm not against adding these as institution types. If we choose this option, Illam and Mana should exclude "house" (so: one institution type per entity), and we should decide on how to create the code name (the short name used in the manuscript identifier). Another option is to add a field for institutions where such local names could be added (so, "type": house; and something like "local name": Illam). @wujastyk, I am happy to hear your input on these questions.

**Comment by wujastyk** [comment]
I like the last idea, with institution=house, type=illam, etc.  It reminds me of XML entities and attributes.   Also, it's expandable.  I have no ide, for example, if there are illam-like places in Orissa, for example.  I do remember seeing courtyard-houses in Chettinad, in Tamil Nadu (but I don't think they had libraries)

Another idea might be to just have a field called "house (including illam, mana etc.)".   Less good, I think.
